frontends/web/src/containers/ProfilePage.js

```diff
@@ -347,6 +347,7 @@ class ProfilePage extends React.Component {
         setSubmitting(false);
       }
     );
+    this.notify();
```

Comment:
Was this left over from your toast changes?

Reply:
Oopsie 😅 I'll fix this right away.